nixos/modules/virtualisation/lxc.nix

```diff
@@ -73,5 +93,26 @@ in
         include ${cfg.package}/etc/apparmor.d/lxc-containers
       '';
     };
+
+    # `lxc-user-nic` needs suid to attach to bridge for unpriv containers.
+    security.wrappers = lib.mkIf cfg.unpriv {
+      lxcUserNet = {
+        source = "${pkgs.lxc}/libexec/lxc/lxc-user-nic";
+        setuid = true;
+        owner = "root";
+        group = "root";
```

Comment:
We should protect this with a group instead of letting all users launch containers. 

- Create a group, maybe named `lxc-user`.
- Set this group to the newly created group
- Change the mode on the wrapper to ensure that world cannot execute
- Update the tests to ensure both allowed and not-allowed paths are tested.